src/api/public/apidocs-new/paths/person_login.yaml

```diff
@@ -0,0 +1,18 @@
+post:
+  summary: Perform a login for some person
```

Comment:
This line says that it's performing a login, but then the description says that it's a fake login endpoint. Should we even document this endpoint? I would not since it seems quite useless to any API user.